tests/PlayerTest.php

```diff
@@ -0,0 +1,122 @@
+<?php
+require_once 'Player.php';
+
+use PHPUnit\Framework\TestCase;
+
+class PlayerTest extends TestCase
+{
+    public function testConstructor(): void
+    {
+        $player = new Player('Lionel Messi', 99, 50, 'FC Barcelona', 'Forward');
+        $this->assertEquals('Lionel Messi', $player->getName());
+        $this->assertEquals(99, $player->getAttack());
+        $this->assertEquals(50, $player->getDefence());
+        $this->assertEquals('FC Barcelona', $player->getClubId());
+    }
+
+    public function testGetId()
+    {
+        $player = new Player('John Doe', 80, 70, '1234', 'Defender', 1);
+        $this->assertEquals(1, $player->getId());
+    }
+
+    public function testGetName()
+    {
+        $player = new Player('John Doe', 80, 70, '1234', 'Defender');
+        $this->assertEquals('John Doe', $player->getName());
+    }
+
+    public function testSetName()
+    {
+        $player = new Player('John Doe', 80, 70, '1234', 'Defender');
+        $player->setName('Jane Doe');
+        $this->assertEquals('Jane Doe', $player->getName());
+    }
+
+    public function testGetAttack()
```

Comment:
Not an actual change request (unless you want to ofc) but most people would say that you don't really need to write unit tests for getters and setters, unless they perform more complex functionality such as your position setter. No need to delete these though.

Reply:
Thank you for the heads up!